### DISCUSSION TITLE
I'm curious why there isn't a "Latent" upscaler option during the hirefix stage. Thank you

### DISCUSSION THREAD

**Discussion by BNP1111** [light-and-ray/sd-webui-replacer#44] [General]
I'm curious why there isn't a "Latent" upscaler option during the hirefix stage. Thank you

like:Latent、Latent (antialiased)，Latent (bicubic)，Latent (bicubic antialiased)，etc

**Comment by light-and-ray** [comment]
They don't exist because there are two img2img operation. Hires fix steps are processed after vae decoding. Add latent upscalers isn't possible without patching original img2img pipeline
